lib/game.rb

```diff
@@ -0,0 +1,83 @@
+class Game
+	attr_accessor :board, :player_1, :player_2
+	attr_reader :win_indexes
+
+	def initialize(player_1 = Players::Computer.new("X"), player_2 = Players::Computer.new("O"), board = Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+		@player_2.game = self 
+		@player_1.game = self 
+	end
+
+	WIN_COMBINATIONS =[
+    	[0, 1, 2],
+    	[3, 4, 5],
+    	[6, 7, 8],
+    	[0, 4, 8],
+    	[2, 4, 6],
+    	[0, 3, 6],
+    	[1, 4, 7],
+    	[2, 5, 8]
+  	]
+
+  	def current_player
+  		if @board.turn_count.even?
```

Comment:
You can use a one line ternary for this method. 

`@board.turn_count.even? ? @player_1 : @player_2`